source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRImageRenderer.mm

```diff
@@ -0,0 +1,106 @@
+//
+//  ACRImageRenderer
+//  ACRImageRenderer.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRImageRenderer.h"
+#import "Image.h"
+#import "SharedAdaptiveCard.h"
+
+@implementation ACRImageRenderer
+
++ (ACRImageRenderer* ) getInstance
```

Comment:
There's a few of these

---
In reply to: [125322115](https://github.com/Microsoft/AdaptiveCards/pull/451#discussion_r125322115) [](ancestors = 125322115)